http-server/pom.xml

```diff
@@ -184,7 +184,7 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcpkix-jdk15on</artifactId>
-            <version>1.68</version>
+            <version>1.70</version>
```

Comment:
Why is this different from the other bouncycastle version in the next commit?

Reply:
Separate artifact with separate versioning. I can migrate to jdk18

Reply:
Done. 1.77 will be used only